### PR TITLE
adding example to rgw line

### PR DESCRIPTION
**Pull request by biohazd**
adding example to rgw line

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard backend`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by varshar16** [comment]
Please sign off your commit (`git commit --amend -s`) and prefix the commit title with `doc/cephadm:`.
Refer the [submitting patches doc](https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst).

**Comment by varshar16** [comment]
jenkins render docs

**Comment by sebastian-philipp** [comment]
I guess you'll still need to find a way to add a signed-off-by line in your commit message.

File: doc/cephadm/adoption.rst [hunk 1]
```diff
@@ -137,6 +137,9 @@ Adoption process
 
    where *<placement>* can be a simple daemon count, or a list of
    specific hosts (see :ref:`orchestrator-cli-placement-spec`).
+   Example::
+   
+     # ceph orch apply rgw myorg us-east-1 --placement="2 myhost1 myhost2"
```
Comment (theanalyst) [inline doc/cephadm/adoption.rst:143]: Can we also clarify what this means, whether it deploys 2 RGWs each to myhost1 & myhost2 or 2 RGWs in total, one each in both the hosts

Comment (sebastian-philipp) [reply doc/cephadm/adoption.rst:143]: Hm. that's why there is a link to https://docs.ceph.com/docs/master/mgr/orchestrator/#placement-specification above. 

Do you think we need to explicitly that that here again?

**Comment by sebastian-philipp** [comment]
ping. please add a sign-off-by line.

**Comment by sebastian-philipp** [comment]
ping. please add a sign-off-by line.

**Comment by sebastian-philipp** [comment]
ping. still requires a `Signed-off-by`-line in the commit message.

**Comment by sebastian-philipp** [comment]
Closed due to a missing DCO. I still would like to merge this PR, so please create another pull request. But this time with a DCO!

**Pull request closed without merging**